ipynb/primary-processing/5-alpha_diversity.md

```diff
@@ -0,0 +1,71 @@
+In this next chapter, we're going to calculate alpha diversity over all of the samples in the meta-analysis. As always, let's first get our environment setup.
+
+```python
+>>> import os
+...
+>>> import americangut.notebook_environment as agenv
+>>> import americangut.util as agu
+...
+>>> import qiime_default_reference as qdr
+...
+>>> chp_path = agenv.activate('5')
+>>> greengenes_tree = qdr.get_reference_tree()
+>>> cpu_count = agenv.get_cpu_count()
+```
+
+And then we'll pull in the files we need for processing.
+
+```python
+>>> ag_pgp_hmp_gg_100nt_biom = agu.get_existing_path(agenv.paths['ag-pgp-hmp-gg-100nt-biom'])
+>>> ag_gg_100nt_biom         = agu.get_existing_path(agenv.paths['ag-gg-100nt-biom'])
+>>> ag_pgp_hmp_gg_cleaned_md = agu.get_existing_path(agenv.paths['ag-pgp-hmp-gg-cleaned-md'])
+```
+
+As well as establish the new paths that we'll be creating.
+
+```python
+>>> ag_pgp_hmp_gg_100nt_1k_multi   = agu.get_new_path(agenv.paths['ag-pgp-hmp-gg-100nt-1k-multiple'])
```

Comment:
it isn't that difficult to add in, however it can be added in later. it is more important to get the processing pipeline back up